### PR TITLE
Added English gasque songs

### PR DESCRIPTION
**Pull request by StelFoog**
Added 'Viking Raiders'
Added 'I take the Snaps'
Added 'In a Cavern'
Added 'Golden Drops'
Added 'What if I had a Little Liquor'
Added 'I Love Punsch'

### CODE REVIEW

File: songs/154_Viking_Raiders.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title: Viking Raiders
+author: Samuel Larsson (2020)
+melody: Spanish Ladies
+tags: [gasque, eng]
+---
+
+Farewell and adieu to you danish ladies
+Farewall and adieu to you lovely danes
```
Comment (Cactooz) [inline songs/154_Viking_Raiders.md:9]:
```suggestion
Farewell and adieu to you lovely Danes
```

File: songs/154_Viking_Raiders.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+For we’re under orders to sail of to England
```
Comment (Cactooz) [inline songs/154_Viking_Raiders.md:10]:
```suggestion
For we’re under orders to sail off to England
```

File: songs/155_I_take_the_Snaps.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+melody: I see the moon
+tags: [snaps, eng]
+---
+
+I take the snaps, the snaps thrills me,
+thrilling as only a snaps can be.
+I don't know where I ought to be
+before I get me snapses three.
+
+Over the mountain over the see
```
Comment (Cactooz) [inline songs/155_I_take_the_Snaps.md:12]:
```suggestion
Over the mountain over the sea
```

File: songs/159_I_Love_Punsch.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+---
+title: I Love Punsch
+author: Samuel Larsson (2020)
+melody: Java Jive
+tags: [punsch, eng]
+---
+
+I love punsch in bites of three
+I love the punsch bottle and it loves me
+Arrack and tea, makes the punsch just for me
```
Comment (Cactooz) [inline songs/159_I_Love_Punsch.md:10]:
```suggestion
Arrack and tea, make the punsch just for me
```

**Review by Cactooz — CHANGES_REQUESTED** [review]
Looks mainly good, add the missing composers and fix the possible typos.

File: songs/159_I_Love_Punsch.md (hunk 4 above)
Comment (StelFoog) [reply songs/159_I_Love_Punsch.md:10]: Here it is implied that the arrack and tea are actively creating the punsch, so this is as intended

**Review by Cactooz — APPROVED** [review]
LGTM

**Pull request merged**